makefiles/tools/targets.inc.mk

```diff
@@ -24,3 +24,8 @@ $(RIOTTOOLS)/mosquitto_rsmb/mosquitto_rsmb:
 
 mosquitto_rsmb: $(RIOTTOOLS)/mosquitto_rsmb/mosquitto_rsmb
 	@make -C $(RIOTTOOLS)/mosquitto_rsmb run
+
+$(RIOTTOOLS)/setsid/setsid:
```

Comment:
Please also add the dependency to the `Makefile` like for `edbg` to make it rebuild on version change.